Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -0,0 +1,168 @@
+import UIKit
+import PayPalNativeCheckout
+import PayPalCheckout
+import PaymentsCore
+import Card
+
+class PayPalViewModel: ObservableObject, PayPalDelegate {
+
+    enum State {
+        case initial
+        case loading(content: String)
+        case mainContent(title: String, content: String, flowComplete: Bool)
+    }
+
+    @Published private(set) var state = State.initial
+
+    private var accessToken = ""
+    private var payPalClient: PayPalClient?
+
+    func getAccessToken() {
+        Task {
+            state = .loading(content: "Getting access token")
+            guard let token = await getAccessToken() else {
+                state = .mainContent(title: "Error", content: "Failed to fetch access token", flowComplete: true)
+
+                return
+            }
+            accessToken = token
+            payPalClient = PayPalClient(config: CoreConfig(accessToken: token, environment: PaymentsCore.Environment.sandbox))
+            payPalClient?.delegate = self
+            state = .mainContent(title: "Access Token", content: accessToken, flowComplete: false)
+        }
+    }
+
+    func retry() {
+        payPalClient = nil
+        accessToken = ""
+        state = .initial
+    }
+
+    func checkoutWithOrder() {
+        startNativeCheckout {
+            await self.payPalClient?.start { createOrderAction in
+                createOrderAction.create(order: OrderRequestHelpers.createOrderRequest())
+            }
+        }
+    }
+
+    func checkoutWithOrderID() {
+        startNativeCheckout {
+            let orderID = try await self.getOrderIDWithFixedShipping()
+            await self.payPalClient?.start { createOrderAction in
+                createOrderAction.set(orderId: orderID)
+            }
+        }
+    }
+
+    func checkoutWithBillingAgreement() {
```

Comment:
We determined in stand-up that we can remove billing agreement & vault flow from the demo. We also can remove the `Order` flow just in favor of keeping the standard OrderID flow. So we'll only remain to test 2 flows in our Demo: orderID & shipping callbacks.